app/controllers/catalog_controller.rb

```diff
@@ -61,6 +61,8 @@ class CatalogController < ApplicationController
     config.show.partials.insert(1, :osd_or_embed)
     config.show.tile_source_field = :content_metadata_image_iiif_info_ssm
 
+    config.show.partials.insert(3, :bibliography)
```

Comment:
(I don't remember what the partials list is, but...) is this being inserted in the middle of the list, or just appended to the end (perhaps suggesting we could do something like `config.show.partials << :bibliography`?